connector/connect/common/pom.xml

```diff
@@ -38,6 +38,18 @@
         <tomcat.annotations.api.version>6.0.53</tomcat.annotations.api.version>
     </properties>
     <dependencies>
+        <dependency>
```

Comment:
Why do we need to add this?

Reply:
For the shared `UdfPacket` (defined in `common`) containing `AgnosticEncoder` related params

Reply:
Sorry I am blind. Yeah that makes sense.